code/computational_geometry/src/bresenham_line/Bresenham_Line_Algorithm.py

```diff
@@ -0,0 +1,72 @@
+# Import graphics library
+from graphics import *
+
+import time
+
+
+def Bresenham_Line(xa, ya, xb, yb):
+
+    dx = abs(xb - xa)
+    dy = abs(yb - ya)
+
+    slope = dy / float(dx)
+
+    x, y = xa, ya
+
+    # creating the window
+    win = GraphWin('Bresenham Line', 600, 480)
+
+    # checking the slope if slope > 1
+    # then interchange the role of x and y
+    if slope > 1:
+        dx, dy = dy, dx
+        x, y = y, x
+        xa, ya = ya, xa
+        xb, yb = yb, xb
+    # initialization of the inital disision parameter
+    p = 2 * dy - dx
+
+    Put_Pixle(win, x, y)         # Plot Pixels To Draw Line
+
+    for k in range(2, dx):
+        if p > 0:
+            y = y + 1 if y < yb else y - 1
+            p = p + 2 * (dy - dx)
+        else:
+            p = p + 2 * dy
+        x = x + 1 if x < xb else x - 1
+
+        time.sleep(0.01)          # delay for 0.01 secs
+        PutPixle(win, x, y)       # Plot Pixels To Draw Line
+
+
+def Put_Pixle(win, x, y):
+
+    """ Plot A Pixle In The Windows At Point (x, y) """
```

Comment:
Pixel, not pixle. Window, not windows. Also only first word should be capitalized. No spaces between quotes and letters either.